pkg/issuer/acme/dns/digitalocean/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 Daniel Beal
```

Comment:
Where did this license come from? This package should be apache licensed with attribution to 'the Jetstack cert-manager contributors' (as per the file headers).

Reply:
I removed it, which I think is the right thing to do (since the headers cover license)